examples/example_hurdles.py

```diff
@@ -86,62 +81,34 @@ def objective_two(individual):
     if not individual.fitness_is_None():
         return individual
 
-    n_function_evaluations = 100000
+    n_function_evaluations = 1000000
 
     np.random.seed(1234)
-    values = np.random.uniform(-4, 4, n_function_evaluations)
+    x = np.random.uniform(-4, 4, n_function_evaluations)[100:]
 
-    f = individual.to_func()
-    loss = 0
-    for x in values[100:]:
-        # the callable returned from `to_func` accepts and returns
-        # lists; accordingly we need to pack the argument and unpack
-        # the return value
-        y = f(x)
-        loss += (f_target(x) - y) ** 2
+    f = individual.to_numpy()
+    y = f(x)
+    loss = np.mean((f_target(x) - y) ** 2)
 
-    individual.fitness = -loss / n_function_evaluations
+    individual.fitness = -loss
 
     return individual
 
 
 # %%
-# Next, we set up the evolutionary search. We first define the
-# parameters for the population, the genome of individuals, and the
-# evolutionary algorithm.
-
-
-population_params = {"n_parents": 1, "seed": 8188211}
-
-genome_params = {
-    "n_inputs": 1,
-    "n_outputs": 1,
-    "n_columns": 12,
-    "n_rows": 1,
-    "levels_back": 5,
-    "primitives": (cgp.Add, cgp.Sub, cgp.Mul, cgp.ConstantFloat),
-}
-
-# %%
-# We define the upper percentile of individuals which are evaluated on
-# the (n+1)th objective by a list of numbers between 0 and 1.
-ea_params = {
-    "n_offsprings": 4,
-    "mutation_rate": 0.03,
-    "n_processes": 1,
-    "hurdle_percentile": [0.5, 0.0],
-}
+# Next, we set up the evolutionary search. We first define the parameters of the
+# evolutionary algorithm. We define the upper percentile of individuals which
+# are evaluated on the (n+1)th objective by a list of numbers between 0 and 1
+ea_params = {"hurdle_percentile": [0.75, 0.0]}
```

Comment:
Just out of curiosity: Why the percentile change?

Reply:
just to make the runtime difference a bit more obvious. i thought i'd sneak it in with this PR ;)